src/components/Storyblok/SbSection.tsx

```diff
@@ -47,35 +49,39 @@ export const SbSection = ({
     pb={paddingBottom}
     className="su-relative su-overflow-hidden"
   >
-    {superhead && (
-      <Text size={2} leading="tight" font="serif" className="su-cc su-mb-08em">
-        {superhead}
-      </Text>
-    )}
-    {heading && (
-      <FlexBox className="su-relative su-rs-mb-6">
+    {(heading || superhead) && (
+      <FlexBox className="su-rs-mb-6">
         {barColorValue && (
           <div className={cnb(
             'su-block su-w-8 md:su-w-20 lg:su-w-40',
             accentBgColors[paletteAccentColors[barColorValue]],
           )}
           />
         )}
-        {heading && (
-          <Heading
-            as={headingLevel}
-            size="splash"
-            leading="none"
-            uppercase
-            font="druk"
-            className={cnb(
-              'su-cc su--mt-[0.14em] su-mb-0 su-whitespace-pre-line su-w-full 3xl:su-max-w-3/5',
-              barColorValue ? 'su--ml-8 md:su--ml-20 lg:su--ml-40' : '',
-            )}
-          >
-            {heading}
-          </Heading>
+        <div className={cnb(
+          'su-cc su-whitespace-pre-line su-w-full 3xl:su-max-w-3/5',
+          barColorValue ? 'su--ml-8 md:su--ml-20 lg:su--ml-40' : '',
+          superhead ? '' : 'su--mt-05em',
         )}
+        >
+          {superhead && (
+            <Text size={2} leading="tight" font="serif" aria-hidden>
+              {superhead}
+            </Text>
+          )}
+          {heading && (
+            <Heading
+              as={headingLevel}
+              size="splash"
+              leading="none"
+              uppercase
+              font="druk"
+              className="su-mb-0"
+            >
+              {superhead && <SrOnlyText>{`${superhead}:`}</SrOnlyText>}{heading}
```

Comment:
Previously recommended by SODA, if we have a superhead above a heading, aria-hidden the visible superhead, and add the superhead as SR only text to the front of the heading.

Reply:
Good to know. This would be a good little bite of info to bring to a dev chat